src/archive.e

```diff
@@ -258,6 +257,60 @@ feature -- Archiving
 			end
 		end
 
+	add_node_entry (a_location: PATH)
```

Comment:
helper routine ... maybe easier to use than knowing which *_ARCHIVABLE object one need to create.